doc/script_commands.txt

```diff
@@ -5389,7 +5389,8 @@ and theirs val1, val2, val3, and val4 usage in source.
 perform a complete removal of all statuses (although permanent ones will re-apply).
 
 'sc_end_class' works like 'sc_end' but will remove all status effects from any learned
-skill on the invoking character.
+skill on the invoking character. if job_id was given it will end the effect for that job.
+if the job was 0 it will return the <job_id> for the invoking character
```

Comment:
I'd rather have it like if you specify a class it will take that value for granted(maybe a small validation, but nothing more).
This would also be better since novice is also job class 0, if I do not remember wrong here.